submissions/tab-tracker/rules.json

```diff
@@ -0,0 +1,62 @@
+[
+    {
+        "id": 1,
+        "condition": {
+            "urlFilter": "harker.org",
```

Comment:
I assume these are websites that are automatically tracked? What if someone wants to change the rules or remove some?